vector/src/main/res/values/donottranslate.xml

```diff
@@ -16,4 +16,10 @@
 
     <!-- onboarding english only word play -->
     <string name="cut_the_slack_from_teams" translatable="false">Cut the slack from teams.</string>
+
+    <!--  WIP -->
+    <string name="ftue_account_created_personalize" translatable="false">Personalise profile</string>
```

Comment:
Default is EN-US so it has to be
```suggestion
    <string name="ftue_account_created_personalize" translatable="false">Personalize profile</string>
```

Reply:
:+1:  will also let design/product know 06f2ca1379c0fb35fc6a3f5b0309cd6e7e4f3470